### DISCUSSION TITLE
Question about Android version

### DISCUSSION THREAD

**Discussion by stephanpdx** [zeapo/Android-Password-Store#3225] [General]
I tried installing this app on my phone using the release channel link but got the following message on Google Play Store: "This app isn't available for your device because it was made for an older version of Android." I was just wondering whether you are aware of this issue and whether there are plans for an updated version, thanks!

Best,
Stephan

**Comment by creepiepanda** [comment]
I suspect they are working on a newer version quite heavily since there's lots of activity in the project and there's a pre-release snapshot build for version 2.0.0 under releases

**Comment by msfjarvis** [comment]
I'm quite aware, you can find the relevant answer in https://github.com/android-password-store/Android-Password-Store/issues/2417.
